tools/autograd/derivatives.yaml

```diff
@@ -743,8 +743,8 @@
   self: mul_tensor_backward(grad, at::scalar_to_tensor(other), self.scalar_type())
 
 - name: mv(Tensor self, Tensor vec) -> Tensor
-  self: grad.ger(vec)
```

Comment:
cc @zou3519 does this fix the perf issues you were seeing by side effect because it makes the vec contiguous?

Reply:
Natalia actually fixed the perf issues by fixing a bug in mv so there is no problem anymore: https://github.com/pytorch/pytorch/pull/44681